src/components/ShoppingLists/ShoppingLists.js

```diff
@@ -0,0 +1,245 @@
+import React, { useState, useEffect } from "react";
+import Text from "@commercetools-uikit/text";
+import { GRAPHQL_TARGETS } from "@commercetools-frontend/constants";
+import { fetchShoppingLists } from "./queries.graphql";
+import {
+  createShoppingList,
+  deleteShoppingList,
+  updateShoppingListName,
+} from "./mutations.graphql";
+import DataTableManager from "@commercetools-uikit/data-table-manager";
+import DataTable from "@commercetools-uikit/data-table";
+import PrimaryButton from "@commercetools-uikit/primary-button";
+import { FormModalPage } from "@commercetools-frontend/application-components";
+import Spacings from "@commercetools-uikit/spacings";
+import TextField from "@commercetools-uikit/text-field";
+import TextInput from "@commercetools-uikit/text-input";
+
+import {
+  useShowApiErrorNotification,
+  useShowNotification,
+} from "@commercetools-frontend/actions-global";
+import { DOMAINS } from "@commercetools-frontend/constants";
+import { useFormik } from "formik";
+
+import { useQuery, useMutation } from "@apollo/client/react";
+
+import IconButton from "@commercetools-uikit/icon-button";
+import {
+  CheckInactiveIcon,
+  EditIcon,
+  CheckBoldIcon,
+} from "@commercetools-uikit/icons";
+
+const target = GRAPHQL_TARGETS.COMMERCETOOLS_PLATFORM;
+
+const ShoppingLists = () => {
+  const [formVisibility, setFormVisibility] = useState(false);
+  const [updateVisibility, setUpdateVisibility] = useState({
+    status: false,
+    rowId: "",
+    version: "",
+    name: "",
+  });
+  const showApiErrorNotification = useShowApiErrorNotification();
+  const showSuccessNotification = useShowNotification();
+  const { error, data, loading } = useQuery(fetchShoppingLists, {
+    context: { target },
+  });
+  const refetch = {
+    // Then re-run
+    refetchQueries: [{ query: fetchShoppingLists, context: { target } }],
+  };
+  const [saveShoppingList] = useMutation(createShoppingList, refetch);
+  const [updateShoppingList] = useMutation(updateShoppingListName, refetch);
+  const [delShoppingList] = useMutation(deleteShoppingList, refetch);
+  const cols = [
+    {
+      key: "id",
+      label: "id",
+      renderItem: (row) => (row?.id ? row.id : ""),
+    },
+    {
+      key: "version",
+      label: "version",
+      renderItem: (row) => (row?.version ? row.version : ""),
+    },
+    {
+      key: "name",
+      label: "Name",
+      renderItem: (row) => {
+        if (updateVisibility.status && updateVisibility.rowId == row.id)
+          return (
+            <TextInput
+              value={updateVisibility.name}
+              onChange={(event) =>
+                setUpdateVisibility({
+                  ...updateVisibility,
+                  name: event.target.value,
+                })
+              }
+            />
+          );
+
+        return row?.nameAllLocales ? row?.nameAllLocales[0]?.value : "";
```

Comment:
You might want to look at the l10n transformers to help rendering a localized field.

https://github.com/commercetools/merchant-center-application-kit/tree/main/packages/l10n